operator/version/version.go

```diff
@@ -15,13 +15,17 @@
 package version
 
 import (
+	"time"
+
 	pkgversion "istio.io/istio/operator/pkg/version"
 	buildversion "istio.io/pkg/version"
 )
 
 const (
 	// OperatorCodeBaseVersion is the version string from the code base.
 	OperatorCodeBaseVersion = "1.17.0"
+	OperatorEOLYear         = 2023
```

Comment:
@ericvn how can we make sure this is updated?

Reply:
We do, just want to make sure we also notice to change this

Reply:
We can add something into the release-builder code as well as the over-arching issue to make sure this happens as part of the updates.